README.md

```diff
@@ -1,33 +1,108 @@
 
-## Assumptions
+> **Warning:**
+> The secrets are stored in plain text and are not secure. Do not use for public
+> accessible systems.
 
-* the 1st blockdevice is available as ``/dev/sda`` or ``/dev/nvme0n1``
+## Requirements
 
-## Download
+* The first blockdevice is available as `/dev/sda` or `/dev/nvme0n1`.
+* You need to have at least a second blockdevice which is from the same type as the first, empty and usable.
+* USB stick with at least 2GB capacity.
+* CPU: 16 core, 32 threads
+* RAM: 128 GB
+* Storage: 2x 1 TB
+* Network: 2 network interfaces
 
-| :zap: When booting from this image, all data on the hard disks will be destroyed without confirmation. |
-|--------------------------------------------------------------------------------------------------------|
+## Download the Cloud in a Box image
 
-* https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-1.iso (with /dev/sda)
-* https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-2.iso (with /dev/nvme0n1)
+> **Warning:** ⚡ When booting from this image, all data on the hard disks will be destroyed without confirmation.
 
-## Usage
+* <https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-1.iso> (with /dev/sda)
+* <https://minio.services.osism.tech/node-image/ubuntu-autoinstall-cloud-in-a-box-2.iso> (with /dev/nvme0n1)
 
-* Copy image to USB stick
-* Boot from USB stick
-* Installation is performed, system shuts down afterwards
-* Remove USB stick and start system
-* Deloyment is performed, system shuts down afterwards
-* System is ready for use, by default DHCP is tried on
-  the 1st network device
-* Login via ``dragon`` and ``password``
+## Getting started
+
+Download the Cloud in a Box image from the link above.
+
+Use a tool to create a bootable USB stick with the Cloud in a Box image.
+
+Boot from the USB stick.
+
+The instalation will start and take a few minutes. After that the system will shutdown.
+
+Remove the USB stick and restart the system.
+
+The deployment will start. This takes some time and the system will shutdown when the deployment is finished.
+This step can take at least one hour or more.
+
+System is ready for use, by default DHCP is tried on the first network device.
+
+Login via ssh. Use the user **dragon** with the password **password**.
+
+```bash
+ssh dragon@IpFromYourServer
+```
+
+Copy the `/home/dragon/wireguard-client.conf` file to your device. This is necessary for using the web endpoints on your device.
+Rename the wireguard config file to something like `ciab.conf`.
+> **Note:**
+> Be aware that the name does not contain any special characters!
+
+```bash
+scp dragon@IpFromYourServer:/home/dragon/wireguard-client.conf /home/ubuntu/ciab.conf
+```
+
+Install wireguard on your device, if you have not done this before. For instructions how to do it on your device, please have
+a look on the documentation of your used distribution.
```

Comment:
Suggestion to add:
Note: If you want to connect from multiple client devices simultaneously, ensure that the ``Address = `` statements in the ``ciab.conf`` files assign a different virtual IP address.

Reply:
As the documentation shouldn´t be overloaded with informations, this is out of scope. The documentation should show only one generic way for all. More ways will confuse beginners.